kubetest/util.go

```diff
@@ -65,7 +65,11 @@ func insertPath(path string) error {
 // Essentially curl url | writer
 func httpRead(url string, writer io.Writer) error {
 	log.Printf("curl %s", url)
-	r, err := http.Get(url)
+	// TODO: we should probably create only one Transport and reuse it for all calls
```

Comment:
Seems awfully short sighted. How would you review this kind of thing in someone else's PR?